velox/dwio/parquet/writer/Writer.cpp

```diff
@@ -75,6 +75,10 @@ class ArrowDataBufferSink : public arrow::io::OutputStream {
 
  private:
   std::unique_ptr<dwio::common::DataSink> sink_;
+  // growRatio_ parameter determines the growth factor used when invoking
```

Comment:
This comment belongs to method-level comments of the constructor. It should not talk about default value since there is none.

Reply:
Updated.

Reply:
I don't see this comment addressed.

Reply:
Sorry i may not have understood you. I have removed the default value comment. Do you mean to delete this comment to `growRatio_`?

Reply:
> This comment belongs to method-level comments of the constructor.

I was trying to say that this comment needs to be moved to the constructor. Users of this class look at public methods first, hence, important comments need to be there.

```
@param Growth factor used...
```

Reply:
Thanks for the detailed explanation. I have updated. Can you help to review again?